www_src/components/select/select.less

```diff
@@ -0,0 +1,12 @@
+.select {
+  -webkit-appearance: none;
```

Comment:
I seem to recall we did auto-prefixing, so the -moz- and -webkit- prefixes should be unnecessary